src/coldcard.js

```diff
@@ -29,33 +29,27 @@ import {
   INFO,
   ERROR,
 } from "./interaction";
-import {
-  P2SH,
-  P2SH_P2WSH,
-  P2WSH,
-} from 'unchained-bitcoin';
+import { P2SH, P2SH_P2WSH, P2WSH } from "unchained-bitcoin";
```

Comment:
don't think this file needs to be touched

Reply:
Seems introducing it by rebasing master.